.github/workflows/tld-update.yml

```diff
@@ -0,0 +1,45 @@
+name: tld-update
+on:
+  schedule:
+    # Run every hour, at the 15 minute mark. E.g.
+    # 2020-11-29 00:15:00 UTC, 2020-11-29 01:15:00 UTC, 2020-11-29 02:15:00 UTC
+    - cron:  '15 * * * *'
+jobs:
+  psl-gtld-update:
+    name: Check for TLD data updates
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+
+      - name: Set current date as env variable
+        run: echo "NOW=$(date +'%Y-%m-%dT%H:%M:%S %Z')" >> $GITHUB_ENV
+
+      - name: Run patchnewgtlds
+        run: tools/patchnewgtlds
+
+      - name: Create pull-request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: "${{ secrets.BOT_PAT }}"
+          push-to-fork: tld-update-bot/list
+          commit-message: "util: gTLD data autopull updates for ${{ env.NOW }}"
+          title: "util: gTLD autopull updates for ${{ env.NOW }}"
+          body: "Public suffix list gTLD data updates from `tools/patchnewgtlds` for ${{ env.NOW }}."
+          committer: "GitHub <noreply@github.com>"
+          author: "GitHub <noreply@github.com>"
+          labels: tld-update
+          branch: psl-gtld-update
+          delete-branch: true
```

Comment:
In the context of the discussion at https://github.com/zmap/zlint/issues/515#issuecomment-743776240 , does this become easier if we change the default?

And we'd also be affected by the lack of testing runs, right? Or does the fact that we're still on Travis and have not yet migrated to GH Actions make this largely moot until we do?

Reply:
> In the context of the discussion at zmap/zlint#515 (comment) , does this become easier if we change the default?

The situation in this repo is slightly different because I chose to keep the separate bot account + to have the PRs opened from a fork. The branch the bot creates will be on the fork repo, whereas in ZLint there is no bot presently and the branch is created on the main repo.

> And we'd also be affected by the lack of testing runs, right? Or does the fact that we're still on Travis and have not yet migrated to GH Actions make this largely moot until we do?

Similar answer here: using the separate bot account makes this repo's implementation a little bit more complex but will avoid the problem related to testing runs in the future if CI is moved from Travis. 

You're correct that it's largely moot until the Travis bits have been migrated - no matter how the PR is created those should continue to work the same.

Reply:
D'oh, resolving this, I *completely* missed the discussion when you opened the PR.